recipes/cccl/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/lib/cmake
+mkdir -p ${PREFIX}/include
+
+mv -v thrust/thrust/cmake ${PREFIX}/lib/cmake/thrust
+mv -v cub/cub/cmake ${PREFIX}/lib/cmake/cub
+cp -rv libcudacxx/lib/cmake/libcudacxx ${PREFIX}/lib/cmake
+
+cp -rv thrust/thrust ${PREFIX}/include
+cp -rv cub/cub ${PREFIX}/include
+cp -rv libcudacxx/include/cuda ${PREFIX}/include
+cp -rv libcudacxx/include/nv ${PREFIX}/include
```

Comment:
Wonder if there is a CMake install approach we can use here in the future

Reply:
This can be reworked with CCCL 2.2 which releases from the new monorepo. Let’s keep the status quo for now since this is just renaming a package we have already tested/used a bit and trust.